modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinConnectionSelfTest.java

```diff
@@ -85,12 +85,24 @@
     private static final String LOCALHOST = "127.0.0.1";
 
     /** URL. */
-    private String url = bestEffortAffinity ?
+    private String url = affinityAwareness ?
         "jdbc:ignite:thin://127.0.0.1:10800..10802" :
         "jdbc:ignite:thin://127.0.0.1";
 
+    /** URL with best effort affinity flag. */
+    private String urlWithBestEffortAffinityFlag = url +
+        (affinityAwareness ?
```

Comment:
it can be simplified to "?affinityAwareness=" + affinityAwareness;

Reply:
fixed